osf/models/node.py

```diff
@@ -241,20 +238,6 @@ class AbstractNode(DirtyFieldsMixin, TypedModel, AddonModelMixin, IdentifierMixi
         'contributors': '_contributors',
     }
 
-    CATEGORY_MAP = {
-        'analysis': 'Analysis',
-        'communication': 'Communication',
-        'data': 'Data',
-        'hypothesis': 'Hypothesis',
-        'instrumentation': 'Instrumentation',
-        'methods and measures': 'Methods and Measures',
-        'procedure': 'Procedure',
-        'project': 'Project',
-        'software': 'Software',
-        'other': 'Other',
-        '': 'Uncategorized',
-    }
-
```

Comment:
lots of these fields moved to their own mixins with field name, logging method, and method for setting it.